modules/ROOT/partials/commands/fullscreen-cmds.adoc

```diff
@@ -1,7 +1,7 @@
 [cols="1,3",options="header"]
 |===
 |Command |Description
-|mceFullScreen |Toggles full screen mode.
+|mceFullScreen |Toggles full screen mode when the editor is in Classic mode.
```

Comment:
```suggestion
|mceFullScreen |Toggles fullscreen mode
```

Reply:
I think with all of the other admonitions of fullscreen only being supported in classic mode, then adding "when the editor is in Classic mode." is overkill so agree with Ben